assets/samplesheet.csv

```diff
@@ -1,3 +1,7 @@
-sample,fastq_1,fastq_2
-SAMPLE_PAIRED_END,/path/to/fastq/files/AEG588A1_S1_L002_R1_001.fastq.gz,/path/to/fastq/files/AEG588A1_S1_L002_R2_001.fastq.gz
-SAMPLE_SINGLE_END,/path/to/fastq/files/AEG588A4_S4_L003_R1_001.fastq.gz,
+species_dir,assembly_name
+25g/data/echinoderms/Asterias_rubens,eAstRub1.3
```

Comment:
Are these relative file paths? Or is there a script that attaches this to the `/lustre/scratch124/tol/projects` part? I could not find that in your scripts/modules.

Reply:
It can be either relative or absolute. If a relative path, it's relative to where the pipeline is run from (and `--outdir` only contains `pipeline_info`).
In production, I envisage these will be absolute paths.

Reply:
Absolute paths are safer definitely.